src/otx/algorithms/detection/task.py

```diff
@@ -443,13 +445,20 @@ def _add_predictions_to_dataset(
         process_saliency_maps=False,
         explain_predicted_classes=True,
         use_ellipse_shapes=False,
+        show_predictions=True,
     ):
         """Loop over dataset again to assign predictions. Convert from MMDetection format to OTX format."""
+        if show_predictions:
```

Comment:
is `show_prediction` related this PR?

Reply:
no, it's not related, but I found it useful to compare real results between PyTorch and OpenVINO during model validation.

Reply:
I think that's not use case for user. How about removing it?

Reply:
We should remove this kind of debug codes